modules/electromagnetics/test/tests/auxkernels/azimuthal_Faradays_law/scalar_azim_magnetic_time_deriv_convergence.py

```diff
@@ -0,0 +1,66 @@
+# Script to run spatial convergence study on microwave_heating.i
+
+import mms
+import sympy
+
+import mooseutils
+import copy
+import pandas
+
+import pyhit
+import moosetree
+
+x_pp = 'h'
+y_pp = ['error_real', 'error_imag','error_azim_dB_dt_scalar']
+
+x = []
+y = [ [] for _ in range(len(y_pp)) ]
+
+
+runs = 6
+
+for i in range(0, runs):
+    mpi = None
+    console = True
+
+    # Read the file
+    root = pyhit.load('scalar_azim_magnetic_time_deriv.i')
+
+    # Locate and modify "x_max" parameter for the mesh
+    mesh = moosetree.find(root, func=lambda n: n.fullpath == '/Mesh/gmg')
+    mesh["nx"] = 10*2**i
+    mesh["ny"] = 10*2**i
+
+    # Write the modified file
+    pyhit.write("scalar_azim_magnetic_time_deriv.i", root)
+
+    input_files = ['scalar_azim_magnetic_time_deriv.i']
+    cli_args = ['-i'] + input_files
+    a = copy.copy(cli_args)
+
+    out = mooseutils.run_executable('/Users/dechcs/projects-EM-update/moose/modules/electromagnetics/electromagnetics-opt', *a, mpi=mpi, suppress_output=not console)
```

Comment:
This won't work. We have MMS test capabilities and should avoid custom scripts like this

Reply:
Good eye and thanks for the catch! I just pushed a fix.